ansible/roles/flannel/templates/flanneld.j2

```diff
@@ -17,4 +17,4 @@ FLANNELD_ETCD_KEYFILE="{{ flannel_etcd_key_file }}"
 
 # Any additional options that you want to pass
 # By default, we just add a good guess for the network interface on Vbox.  Otherwise, Flannel will probably make the right guess.
-FLANNEL_OPTIONS="{{ flannel_opts | default('') }}"
+FLANNEL_OPTIONS="{{ flannel_options | default('') }}"
```

Comment:
Can't find any ``flannel_options`` in ``defaults/main.yml``. Think you can add the following line into it?

```yaml
#flannel_options: --etcd-prefix=/coreos.com/network
```

So it is obvious where to set the variable.